### DISCUSSION TITLE
xmake 每次只显示一两条警告信息

### DISCUSSION THREAD

**Discussion by twinstar6980** [xmake-io/xmake#4224] [General]
我需要排查项目中的 warning 信息，当 xmake 编译时只显示一两条 warning ，我得 fix 掉一处 warning 后再次编译才能看到下一条 warning ，有没有命令行选项来让 xmake 显示所有 warning ？

![WA](https://github.com/xmake-io/xmake/assets/37923060/e236ecdb-e382-4c15-9768-a94188138761)

**Comment by waruqi** [comment]
`xmake -v`

**Comment by twinstar6980** [comment]
试过了，也是每次只显示少量 warning

![waaa](https://github.com/xmake-io/xmake/assets/37923060/fada28db-5a5b-4b1c-b287-48cd416939b1)

**Reply by waruqi** [reply]
`xmake -D`

**Comment by twinstar6980** [comment]
ok了，谢谢作者
